src/data/markdown/docs/01 guides/02 Using k6/10 Protocols/03 gRPC.md

```diff
@@ -0,0 +1,15 @@
+---
+title: 'gRPC'
+excerpt: ''
+---
+
+## Overview
+
+[gRPC](https://grpc.io/) is a light-weight open-source RPC framework. It was originally developed by Google, with 1.0
```

Comment:
> light-weight

[Citation needed] 😄

Also, I think the correct spelling is lightweight, no hyphen.

Reply:
yeah, will fix the spelling (add a comment in https://github.com/loadimpact/new.blog.io/pull/83 as well), though lightweight is a relative term :sweat_smile: in terms of code complexity, gRPC is definitely not that, but in terms of performance it probably is